src/core/Icon/Icon.tsx

```diff
@@ -84,24 +84,15 @@ export class Icon extends Component<IconProps> {
     } = withSuomifiDefaultProps(this.props);
     const { className, ariaLabel } = this.props;
 
+    const iconColor =
```

Comment:
This does not take style props into account, color might be defined there as well.

Reply:
It seems that user is not able to give custom color for icons that starts with: `staticIllustration`.

Should the illustration versions support of changing the "main color"? Like currently they have that orange color.

Reply:
I think the colors for static icons are fixed inside the SVG's and are not supposed to be changed. The only thing worth supporting for those that comes to mind is "dark mode", but this is not supported by anything we currently have.

Reply:
I do agree, that the API is kind of confusing since it's possible to define a color that does not have any effect on the result.

Reply:
@J-Kallunki any thoughts on this?

Reply:
As agreed while discussing the issue, the API will remain as is for now.